rdkcentral/Dobby
# Add set-log-level command

### Description
Add `set-log-level` command to DobbyTool for easier debugging

### Test Procedure
Start DobbyDaemon with default log level (milestone)  - `DobbyDaemon --nofork`

Run `DobbyTool set-log-level debug`. DobbyDaemon should now start printing Debug level log messages

### Type of Change

### Requires Bitbake Recipe changes?
